### PR TITLE
Update to Eslint 3.8.1

### PR DESCRIPTION
**Pull request by aju**
Passes in Symfony and WC

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -51,8 +51,6 @@
     "no-invalid-regexp": "error",
     // disallow irregular whitespace outside of strings and comments
     "no-irregular-whitespace": "error",
-    // disallow negation of the left operand of an in expression
```
Comment (aju) [inline .eslintrc]: Deprecated

File: .eslintrc [hunk 2]
```diff
@@ -147,8 +153,6 @@
     "no-multi-spaces": "off",
     // disallow use of multiline strings
     "no-multi-str": "error",
-    // disallow reassignments of native objects
```
Comment (aju) [inline .eslintrc]: Deprecated

File: .eslintrc [hunk 3]
```diff
@@ -351,8 +365,8 @@
     "no-plusplus": "off",
     // disallow use of certain syntax in code
     "no-restricted-syntax": "off",
-    // disallow space between function identifier and application
```
Comment (aju) [inline .eslintrc]: Deprecated

**Comment by kdzwinel** [comment]
hm, have you already tested it on wc/symfony? how many errors are there?

**Comment by aju** [comment]
@kdzwinel 0 errors

**Pull request merged**